gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -217,7 +217,16 @@ protected void submitFlowToDagManagerHelper(String flowGroup, String flowName) {
 
   @Override
   protected void createMetrics() {
-    super.messagesRead = this.getMetricContext().counter(ServiceMetricNames.GOBBLIN_SERVICE_PREFIX + RuntimeMetrics.GOBBLIN_KAFKA_HIGH_LEVEL_CONSUMER_MESSAGES_READ);
+    super.messagesRead = this.getMetricContext().counter(RuntimeMetrics.DAG_ACTION_STORE_MONITOR_PREFIX + "." + RuntimeMetrics.GOBBLIN_KAFKA_HIGH_LEVEL_CONSUMER_MESSAGES_READ);
+    super.queueSizeGauges = new ContextAwareGauge[super.numThreads];
+    for (int i=0; i < numThreads; i++) {
+      // An 'effectively' final variable is needed inside the lambda expression below
+      int finalI = i;
+      this.queueSizeGauges[i] = this.getMetricContext().newContextAwareGauge(
+          RuntimeMetrics.GOBBLIN_KAFKA_HIGH_LEVEL_CONSUMER_QUEUE_SIZE_PREFIX + "-" + i,
+          () -> super.queues[finalI].size());
+    }
```

Comment:
this may not be clicking for me... but why can't this all be replaced by `super.createMetrics()`?

Reply:
Oh oops I overwrote the important piece of the code, but I need to add the `gobblin.service` and `dagActionMonitor` prefix to actually report this metric and be able to distinguish it from any other classes that use `HighLevelConsumer`